### ISSUE TITLE
Redis worker PID shoud not be hardcoded in test suite

### ISSUE THREAD

**Issue by heartsucker** [freedomofpress/securedrop#3606]
## Description

Reporting this here while documenting TODOs (#724).

From `securedrop/tests/conftest.py`:

```python
# TODO: the PID file for the redis worker is hard-coded below.
# Ideally this constant would be provided by a test harness.
# It has been intentionally omitted from `config.py.example`
# in order to isolate the test vars from prod vars.
TEST_WORKER_PIDFILE = '/tmp/securedrop_test_worker.pid'
```

**Comment by heartsucker** [comment]
From `securedrop/tests/utils/env.py`:

```python
# TODO: the PID file for the redis worker is hard-coded below.  Ideally this
# constant would be provided by a test harness.  It has been intentionally
# omitted from `config.py.example` in order to isolate the test vars from prod
# vars.  When refactoring the test suite, the test_worker_pidfile
# test_worker_pidfile is also hard-coded in `manage.py`.
TEST_WORKER_PIDFILE = "/tmp/securedrop_test_worker.pid"
```

**Comment by legoktm** [comment]
Coincidentally this was resolved by @nabla-c0d3 today with 2cb4812f590689628441f21e00c0664e27ff7ed6, which moved the hardcoded filename into a fixture so it's now "provided by a test harness".
